src/dlgMapper.cpp

```diff
@@ -334,6 +334,8 @@ void dlgMapper::slot_switchArea(const int index)
 
 void dlgMapper::updateInfoContributors()
 {
+    return;
```

Comment:
Wait but why?

Reply:
Accidental commit to work around the crash